internal/sbi/producer/nf_management.go

```diff
@@ -248,21 +248,29 @@ func GetNFInstancesProcedure(nfType string, limit int) (*nrf_context.UriList, *m
 		}
 		return nil, problemDetail
 	}
-	logger.NfmLog.Infoln("UL: ", UL)
-	originalUL := &nrf_context.UriList{}
-	if err := mapstructure.Decode(UL, originalUL); err != nil {
-		logger.NfmLog.Errorf("Decode error in GetNFInstancesProcedure: %+v", err)
-		problemDetail := &models.ProblemDetails{
-			Title:  "System failure",
-			Status: http.StatusInternalServerError,
-			Detail: err.Error(),
-			Cause:  "SYSTEM_FAILURE",
+	logger.NfmLog.Infoln("ULs: ", ULs)
+	rspUriList := &nrf_context.UriList{}
+	for _, UL := range ULs {
+		originalUL := &nrf_context.UriList{}
+		if err := mapstructure.Decode(UL, originalUL); err != nil {
+			logger.NfmLog.Errorf("Decode error in GetNFInstancesProcedure: %+v", err)
+			problemDetail := &models.ProblemDetails{
+				Title:  "System failure",
+				Status: http.StatusInternalServerError,
+				Detail: err.Error(),
+				Cause:  "SYSTEM_FAILURE",
+			}
+			return nil, problemDetail
+		}
+		rspUriList.Link.Item = append(rspUriList.Link.Item, originalUL.Link.Item...)
+		if rspUriList.NfType == "" && originalUL.NfType != "" {
```

Comment:
I'm not sure this is a good idea, because this way you always pick the first NF type you encounter, even though there may be different NF types in subsequent documents. Does the specification even mention this field? I can't look it up efficiently because I'm on vacation right now. If it is not mentioned in the specs I would omit NfType` altogether. If not, I would omit it if there are two or more different NF types in the result list.

Reply:
Hi @BENHSU0723 
Please set `omitempty` attribute in the `NFType` field as we discussed yesterday.
Hi @GKnirps 
The specification hasn't mentioned this field, your concern is right.